client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -175,21 +179,20 @@ class CancelPurchaseButton extends Component {
 				/>
 			</Dialog>
 		);
-	}
+	};
 
 	goToCancelConfirmation = () => {
-		const { id } = this.props.purchase,
-			{ slug } = this.props.selectedSite;
+		const { id } = this.props.purchase, { slug } = this.props.selectedSite;
 
 		page( paths.confirmCancelDomain( slug, id ) );
-	}
+	};
 
 	cancelPurchase = () => {
 		const { purchase, translate } = this.props;
 
 		this.toggleDisabled();
 
-		cancelPurchase( purchase.id, ( success ) => {
+		cancelPurchase( purchase.id, success => {
```

Comment:
Yeah I don't think it's a good idea to remove parentheses in that case since `success` now looks like just another function parameter.